Dockerfile

```diff
@@ -62,8 +58,21 @@ RUN apt-get update \
     libunwind8 \
     libuuid1 \
     zlib1g \
+    xz-utils \
   && rm -rf /var/lib/apt/lists/*
 
+# Download and unpack Node.js v10
+RUN mkdir -p /opt \
+  && curl -L https://nodejs.org/dist/v10.16.0/node-v10.16.0-linux-x64.tar.xz -o /opt/node-v10.16.0-linux-x64.tar.xz \
```

Comment:
Little worried about how we keep the version of nodejs you're running here up to date.

Reply:
So, first of all, it does not make it any worse because previously it was Node 4 from Ubuntu :)  Then, this version of Node.js is only used for compiling protos. I suggest we leave both versions (Node.js and google-gax) hardcoded here for now (same way as we do for other stuff in this `Dockerfile` such as `protoc` version, etc.) and update as needed. 

Actually, we'll get rid of artman soon (I hope so) so who cares?